### ISSUE TITLE
About traffic junction Env training and GPU available

### ISSUE THREAD

**Issue by paulbeomK** [sisl/DICG#3]
Hi, I'm sorry to trouble you. 
I am trying to run the code in Traffic Junction (each version) and follow the arguments. 
But I find the result is not right (except easy version) for medium and hard version.  
Finally, the evaluation success rate is just 0 when killed at total env steps (2.5e6 environment steps you already set)
I don't adjust anything about the initial setting.
command: python dicg_de_traffic_runner.py --difficulty 'medium' (or dicg_ce_traffic_runner.py)
                  python dicg_de_traffic_runner.py --difficulty 'hard' (or dicg_ce_traffic_runner.py)
And I don' t know where the error is now. 
Also, Can I train with GPU? (CUDA available)?
I want to seek your help. Thanks!

**Comment by parachutel** [comment]
Hi, thanks for reaching out and being interested in our code.
It seems to be a mistake in setting an overly large default learning rate. I have pushed a fix.

I did a test run for `python dicg_de_traffic_runner.py --difficulty medium`, and after 30 epochs (around 1.7e5 steps) the evaluation success rate could reach 0.45.

```
-----------------------  ----------------
AverageDiscountedReturn     -17.1053
AverageReturn               -20.9577
Entropy                       0.606372
EnvExecTime                  10.6465
EvalAvgReturn               -40.3051
EvalSucessRate                0.45
GradNorm                      0.622601
Iteration                    20
KL                            0.000862986
KLBefore                      0.0009236
LossAfter                    -0.0296079
LossBefore                   -0.0329358
MaxReturn                    -2.882
MinReturn                   -58.443
NumTrajs                   2000
PolicyExecTime               10.8461
ProcessExecTime               0.183713
StdReturn                    10.9585
TotalEnvSteps            168000
dLoss                        -0.00332781
-----------------------  ----------------
```

As for GPU support, current version does not support CUDA. To use CUDA you need to do fair amount of customization in `/dicg/sampler/centralized_ma_on_policy_vectorized_sampler.py` to copy items in `running_paths` to cuda.

**Comment by paulbeomK** [comment]
Hi, Thanks for your help and quick response.
I run the code with the fixed parameter, and can get performance you said on the paper!
